apps/communications/contacts/js/utilities/object.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+(function(exports) {
+
+  exports = exports || {};
+
+  var object = {
+    // Checks if an object fields are empty, by empty means
+    // field is null and if it's an array it's length is 0
+    isEmpty: function (obj, fields) {
```

Comment:
I'm wondering if we should move this to the`ContactsService` that already has a method isEmpty to know if we don't have contacts. But definitely is somehow associated to know something from a mozcontact